Gemfile

```diff
@@ -58,6 +58,9 @@ group(:development, :test) do
   gem 'rdoc', "~> 4.1", :platforms => [:ruby]
   gem 'yard'
 
+  # ronn is used for generating manpages.
+  gem 'ronn', '~> 0.7.3'
```

Comment:
ronn depends on rdiscount which has to build native extensions during installation, so basically you need a compiler, ruby-devel, etc. I think that is ok for CI hosts, e.g. we have the same issue with nokogiri and that work fine. But we may have a problem on Windows hosts, as installing the ruby devkit is a pain. I suspect appveyor completed successfully because they already have the devkit installed. Since we're never going to generate manpages on a windows box, I think we should add `:platforms => [:ruby]` like we do for rdoc. That really means MRI ruby excluding windows, jruby, etc.

Reply:
Not sure if it makes it better or worse, but the reason it worked on Windows is that we don't install that group during CI.

```
bundle install --jobs 4 --retry 2 --without development extra
```